library/alloc/src/collections/btree/set_val.rs

```diff
@@ -2,6 +2,7 @@
 /// Used instead of `()` to differentiate between:
 /// * `BTreeMap<T, ()>` (possible user-defined map)
 /// * `BTreeMap<T, SetValZST>` (internal set representation)
+#[allow(non_camel_case_types)]
 #[derive(Debug, Eq, PartialEq, Ord, PartialOrd, Hash, Clone, Default)]
 pub struct SetValZST;
```

Comment:
```suggestion
pub struct SetValZst;
```
`Zst` is fine in this context.